### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by antoniovazquezblanco**

### CODE REVIEW

**Comment by antoniovazquezblanco** [comment]
It seems that documentation coverage reporting now works. Doxygen is detecting little to no doc but that will improve overtime :)

File: doc/Index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+\mainpage Sigutils documentation
+
+The sigutils library is a digital signal processing library written in C, designed for blind signal analysis and automatic demodulation in GNU/Linux.
+
+You may be able to install sigutils in your operative system using your default package manager. If not, you can build the project form source and install it. For instructions on how to do so, please visit section \ref building.
```
Comment (BatchDrake) [inline doc/Index.md:5]: _operating_ system

**Comment by BatchDrake** [comment]
LGTM

**Comment by antoniovazquezblanco** [comment]
Thanks!

**Pull request merged**